build.py

```diff
@@ -0,0 +1,418 @@
+    Shows a spinning Cursor while the command runs.\n
+    Exits with returncode of subprocess if not equals 0.\n
+    """
+    try:
+        p = None
+        # Register handler to pass keyboard interrupt to the subprocess
+
+        def handler(sig, frame):
+            print(f"================= {name} ABORTED BY USER ====================")
+            if p:
+                p.send_signal(signal.SIGINT)
+            else:
+                raise KeyboardInterrupt.add_note()
+
+        signal.signal(signal.SIGINT, handler)
+        with Spinner():
+            print(f"{name.rstrip('Ee')}ing...")
+            with proc as p:
+                for line in iter(p.stdout.readline, b""):
+                    # Print the line to the console
+                    sys.stdout.buffer.write(line)
+                for line in iter(p.stderr.readline, b""):
+                    # Print the error output to the console
+                    sys.stderr.buffer.write(line)
+                returncode = p.wait()
+                if returncode:
+                    print(
+                        f"===================== {name} FAILED ========================="
+                    )
+                    sys.exit(returncode)
+                else:
+                    print(
+                        f"===================== {name} SUCCESSFUL ====================="
+                    )
+    finally:
+        signal.signal(signal.SIGINT, signal.SIG_DFL)
+
+
+class Build:
+    def __init__(
+        self,
+        openstack: bool,
+        template: str,
+        conda_env: pathlib.Path,
+        provisioning: [str],
+        comment: str,
+        pvt_key: pathlib.Path,
+        ansible_args: str,
+    ):
+        self.openstack = openstack
+        self.template = template
+        self.comment = comment
+        self.pvt_key = pvt_key
+        self.provisioning = provisioning
+        self.ansible_args = ansible_args
+        self.image_name = self.assemble_name()
+        self.image_path = pathlib.Path(DIR_PATH + "/" + self.image_name + ".raw")
+        if conda_env:
+            self.qemu_path = f"{conda_env}/bin/qemu-img"
+            self.openstack_path = f"{conda_env}/bin/openstack"
+            self.packer_path = f"{conda_env}/bin/packer"
+        else:
+            self.packer_path = shutil.which("packer")
+            self.openstack_path = shutil.which("openstack")
+            self.qemu_path = shutil.which("qemu-img")
+
+    def dry_run(self):
+        print(self.assemble_packer_envs())
+        print(self.assemble_packer_build_command())
+        print(self.image_name)
+        print(self.assemble_convert_command())
+        if self.openstack:
+            print(self.assemble_os_command())
+        if self.pvt_key:
+            print(self.assemble_scp_command())
+            print(self.assemble_ssh_command())
+
+    def assemble_packer_init(self):
+        return [
+            f"self.packer_path",
+            f"init",
+            f"{DIR_PATH}/templates",
```

Comment:
```suggestion
            f"{DIR_PATH / 'templates'}",
```

(only if you converted DIR_PATH to a `pathlib.Path` object)